src/bin/agent/CommandStart.ts

```diff
@@ -5,6 +5,7 @@ import type { RecoveryCode } from '../../keys/types';
 import path from 'path';
 import child_process from 'child_process';
 import process from 'process';
+import { getDefaultSeedNodes } from '../utils/parsers';
```

Comment:
This line can be removed because `parseSeedNodes` and `parseNetwork` can be combined together.